### PR TITLE
bug #32 closes , correct bug number done 

### LINKED ISSUE #32: Fix Lintian issue wrong bug number in closes
- Issue type: Bug report 

### Short description

Lintian reports:
`W: systemlock: wrong-bug-number-in-closes l3:#nnnn`

### Environment

 - Operating system: Ubunutu 18.04, travis
 - Software version: 
 - Software source: 

### Steps to reproduce

In the current build;
1. cd into the systemlock-0.1 folder
2. run debuild -uc -us
3. Their will be lintian warnings
Note: In the current build you will get errors; that are solved in pr #26; but for now you can solve it

### Description

See section 4.4. of the Debian Policy manual.
https://www.debian.org/doc/debian-policy/ch-source.html#debian-changelog-debian-changelog

### PR DESCRIPTION
**Pull request by Manas-kashyap**
Fixes #32 
Checklist
I have read the Contribution & Best practices Guide and my PR follows them.
My branch is up-to-date with the Upstream master branch.
I have added necessary documentation (if appropriate)
Added brief description
Changes proposed in this pull request:
Change done in debian/changelog resolving bug number 

Thank you!

### CODE REVIEW

**Review by pranav1698 — CHANGES_REQUESTED** [review]
@Manas-kashyap This pr should contain changes only for the issue #32 
Not for the other pr

**Review by pranav1698 — APPROVED** [review]
LGTM!

**Review by tabesin — CHANGES_REQUESTED** [review]
Pls rebase

File: systemlock-0.1/debian/changelog [hunk 1]
```diff
@@ -0,0 +1,5 @@
+systemlock (0.1-1) unstable; urgency=low
```
Comment (pranav1698) [inline systemlock-0.1/debian/changelog]: Delete this file

**Review by pranav1698 — CHANGES_REQUESTED** [review]

File: systemlock-0.1/debian/changelog (hunk 1 above)
Comment (Manas-kashyap) [reply systemlock-0.1/debian/changelog]: Done

**Review by pranav1698 — APPROVED** [review]

**Review by tabesin — APPROVED** [review]

**Pull request merged**